src/extension/importer/core/src/main/java/org/geoserver/importer/DataStoreFormat.java

```diff
@@ -224,7 +223,7 @@ public DataStore createDataStore(ImportData data) throws IOException {
 
             if (f != null) {
                 Map<String,Serializable> map = new HashMap<String, Serializable>();
-                map.put("url", relativeDataFileURL(Resources.find(f).toURI().toURL().toString(), catalog));
+                map.put("url", relativeDataFileURL(f.toURI().toURL().toString(), catalog));
```

Comment:
The DataUtilities method is slightly more stable then toURI().toURL() on platforms like windows.